Add "Africa" tag and capitalize “Lesson Development” and "Hackathon" tags

ceesu opened the pull request:
Adding some tags for tagathon #73

serahkiburu committed:
Merge pull request #79 from LibraryCarpentry/tagathon-contributions

Tagathon contributions from staff reading party

ceesu committed:
try Africa tag

ceesu committed:
capitalize tags

ceesu committed:
Merge remote-tracking branch 'upstream/master'

serahkiburu reviewed (changes requested):
Hi @ceesu, thanks for your contributions toward Carpentries Tagathon.

[This guide](https://docs.carpentries.org/topic_folders/communications/guides/select-blog-tags.html) has helpful suggestions on standardising tags e.g. Hackathons and sprints fall under `Events`, lesson development and maintenance fall under `Carpentries Lessons`

Using the [list of tags](https://carpentries.org/posts-by-tags) on the central Carpentries blog as a cheatsheet might also help.

ceesu committed:
fix tags based on PR #81

ceesu committed:
tried fixing some of teh tags with quotation marks

ceesu committed:
replaced other tags with quotes

ceesu committed:
Capitalize workshop tag on one post

ceesu commented:
I see, I've updated the tags. Hopefully this is better. Also I think I have fixed the typographic quotation marks to regular quotation marks for the tags with quotes.

serahkiburu commented:
Many thanks @ceesu! I took a short break, but I'm now at my desk and will be merging this shortly. We really appreciate all your input during our Tagathon week.

serahkiburu committed:
Update 2019-01-10-lesson-program-implementation.md

serahkiburu committed:
Update 2019-02-28-news-from-lc-maintainers.md

serahkiburu committed:
Update 2019-03-28-lc-mozilla-global-sprint.md

serahkiburu committed:
Update 2019-04-17-what’s-new-orcid-joint-library-carpentriesorcid-webinar.md

serahkiburu committed:
Update 2019-06-05-may-global-sprint.md

serahkiburu committed:
Update 2019-06-19-hackyhours-zbmed.md
